models/games.models.js

```diff
@@ -1,7 +1,18 @@
 const db = require("../db/connection.js");
+const reviews = require("../db/data/test-data/reviews.js");
```

Comment:
I think this might have been VS Code auto importing - but it CERTAINLY shouldn't be here 👍
Please remove